apps/api/src/services/scrapers/weibo/scraper.py

```
@@ -31,7 +31,7 @@ class WeiboDataProcessor(DataProcessor):
     def __init__(
             self,
             url: str,
-            method: str = "webpage",
+            method: str = "api",
             user_agent: dict = None,
```

Ruff (0.15.6)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)